pkg/cmd/cfg.go

```diff
@@ -40,3 +44,33 @@ func initConfig(configPath string) (*config, error) {
 
 	return cfg, nil
 }
+
+// putCallConfigToEtcd loads the calls config from a specific config file and pushes it to Etcd
+// The function also accepts etcd settings like host and dial timeout.
+// The function will return the Etcd key it had used to put the CallConfig
+// The function may return empty key and an error in case of any errors.
+func putCallConfigToEtcd(ctx context.Context, configPath, etcdURL string, dialTimeoutSeconds int) error {
+	cfg, err := initConfig(configPath)
+
+	if err != nil {
+		return err
+	}
+
+	callConfig := cfg.API.Calls
+
+	callConfigJSON, err := json.Marshal(callConfig)
+
+	if err != nil {
+		return err
+	}
+
+	etcd := NewEtcdHandler(etcdURL, time.Duration(dialTimeoutSeconds*int(time.Second)))
+
+	err = etcd.Put(ctx, "CallConfig", string(callConfigJSON))
+
+	if err != nil {
+		return err
+	}
```

Comment:
@shan-96 lets move this logic into call repo, in this case in this case we'll be able to run validation logic before config upload.